docs/api/event-brokers.rst

```diff
@@ -72,7 +79,7 @@ Here is how you add it using Python code:
     pika_broker = PikaEventBroker('localhost',
                                   'username',
                                   'password',
-                                  queue='rasa_core_events')
+                                  queue='rasa_events')
```

Comment:
the default queue is still `rasa_core_events`, but it makes sense to use this one in the docs for someone setting this up from scratch

Reply:
yeah, i was wondering how this even got set up. bc don't they have to configure the queue in their endpoints? Maybe it's not super clear that that needs to match this one if that is the case

Reply:
@ricwo what do you mean by `default queue` btw? if you don't assign a queue in your endpoints does it create one?

Reply:
that's right, if you don't specify `queue` it uses this the default kwarg